<launch>
	<node pkg="snapshotter" type="snapshotter.py" name="snapshotter_pros" output="screen">
		<param name="camera" value="prosilica" />
		<param name="output" value="snapshot_output" />
	</node>
	
	<node pkg="snapshotter" type="snapshot_saver.py" name="saver" output="screen">
	    <param name="input" value="birdseye_output" />
	    <param name="default_filepath" value="~/snapshots" />
	    <param name="save_camera_info" value="false" />
	    <param name="streaming" value="true" />
	</node>

    <node pkg="snapshotter" type="birdseye_filter.py" name="birdseye"
output="screen">
    <param name="input" value="snapshot_output" />
    <param name="output" value="birdseye_output"/>
    <param name="mode" value="load_h"/>
    <param name="matrix_location" value="/home/stephen/snapshots/H.yaml"/>
</node>

</launch>
